yowsup/demos/cli/layer.py

```diff
@@ -298,6 +299,17 @@ def group_kick(self, group_jid, jids):
             entity = RemoveParticipantsIqProtocolEntity(self.aliasToJid(group_jid), jids)
             self.toLower(entity)
 
+    @clicmd("Remove all participants Group")
+    def group_clean(self, group_jid, jids = None):
+        if self.assertConnected():
+            grupos = RemoveAllParticipantsGroupIqProtocolEntity(self.aliasToJid(group_jid))
```

Comment:
Please rename the variable to "groups" where are usng Enlgish as main language in this repo.